FEM/rf_pcs.cpp

```diff
@@ -13635,73 +13636,64 @@ void CRFProcess::UpdateTransientBC()
 			/// GIS_shape_head[4]:  cellsize
 			/// GIS_shape_head[5]:  NONDATA_value
 			double* g_para = precip->GIS_shape_head;
-			long size = (long)(g_para[0] * g_para[1]);
+			std::size_t ncols = static_cast<std::size_t>(g_para[0]);
+			std::size_t nrows = static_cast<std::size_t>(g_para[1]);
+			std::size_t size = ncols * nrows;
+			const double cell_area = g_para[4] * g_para[4];
 			vector<double> cell_data_p(size);
 			vector<double> cell_data_v(size);
-			for (i = 0; i < size; i++)
+			for (std::size_t i = 0; i < size; i++)
```

Comment:
`i` is used in line 13620. please check if there is any problem

Reply:
There is no problem in computation but this variable name causes confusion. I have changed the source according in the new commit.